aws-py-assume-role/create-role/venv/pyvenv.cfg

```diff
@@ -0,0 +1,3 @@
+home = /Users/xuanhezhou/examples/aws-py-assume-role/create-role/venv/bin
```

Comment:
This file (`aws-py-assume-role/create-role/venv/pyvenv.cfg`)  should be deleted.